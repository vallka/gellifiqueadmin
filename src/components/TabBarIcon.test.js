import React from 'react';
import {shallow} from 'enzyme';
import TabBarIcon from './TabBarIcon';


describe('TabBarIcon Rendering', () => {
    it('TabBarIcon Rendering should match to snapshot', () => {
        const component = shallow(<TabBarIcon />)
        expect(component).toMatchSnapshot()
    });
    it('TabBarIcon focused Rendering should match to snapshot', () => {
        const component = shallow(<TabBarIcon focused={true} />)
        expect(component).toMatchSnapshot()
    });
});

