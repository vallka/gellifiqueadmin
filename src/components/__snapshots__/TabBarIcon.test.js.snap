// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabBarIcon Rendering TabBarIcon Rendering should match to snapshot 1`] = `
<Icon
  allowFontScaling={false}
  color="#333"
  size={26}
  style={
    Object {
      "marginBottom": -3,
    }
  }
/>
`;

exports[`TabBarIcon Rendering TabBarIcon focused Rendering should match to snapshot 1`] = `
<Icon
  allowFontScaling={false}
  color="#d73672"
  size={26}
  style={
    Object {
      "marginBottom": -3,
    }
  }
/>
`;
